### PR TITLE
[SYCL][Matrix] Remove xfail from test that passes now

### PR DESCRIPTION
**Pull request by YuriPlyakhin**

### CODE REVIEW

File: SYCL/Matrix/XMX8/element_wise_all_ops_int8_packed.cpp
Comment (YuriPlyakhin) [inline SYCL/Matrix/XMX8/element_wise_all_ops_int8_packed.cpp]:
```suggestion
// This test stores the matrix B that is VNNIed (packed).
```

**Review by dkhaldi — COMMENTED** [review]
@YuriPlyakhin the changes look good to me.
What are waiting on to make it ready to be reviewed and merged?

**Review by sergey-semenov — APPROVED** [review]

**Comment by YuriPlyakhin** [comment]
@intel/llvm-reviewers-runtime , could you please merge?

**Comment by aelovikov-intel** [comment]
@intel/llvm-gatekeepers have the merge rights, not @intel/llvm-reviewers-runtime .

**Pull request merged**
